packages/types/types/test.tsx

```diff
@@ -62,6 +68,53 @@ const myBlockLink = (
     dsRef={() => 'ima ref'}
   />
 )
+// $ExpectType Element
+const myButton = (
+  <Button
+    variation={'fill'}
```

Comment:
n-b curly braces around strings are unnecessary

Reply:
Will fix! Wasn't sure as it seemed like the standard in this file. Thanks!